File: api/grpc/todo/ping.go
package todo

import (
	"context"
	"log"
)

type PingHandler struct{}

func (c *PingHandler) Ping(ctx context.Context, in *PingRequest) (*PingResponse, error) {
	log.Printf("Received: %v", in.Data)
	return &PingResponse{Data: "Data: " + in.Data}, nil
}

File: api/grpc/todo/ping.py
import grpc
import os
import ping_pb2_grpc
import ping_pb2

channel = grpc.insecure_channel('0.0.0.0:8080', options=(('grpc.enable_http_proxy', 0),))
stub = ping_pb2_grpc.PingServiceStub(channel)

req = ping_pb2.PingRequest(data="1st ping")
res = stub.Ping(req)
print(res)